src/seedu/addressbook/Main.java

```diff
@@ -80,6 +82,12 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
+                checkStorageFile();
```

Comment:
Not the best place to check for the exception. Should be save() in StorageFile.java